data/avro-schema/src/main/resources/avro/net/corda/data/services/data/TokenClaimQueryResult.avsc

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "record",
+  "name": "TokenClaimQueryResult",
+  "namespace": "net.corda.data.services.data",
+  "doc": "Represents the result of a claim query, sent back to the flow",
+  "fields": [
+      {
+        "name": "requestContext",
```

Comment:
You shouldn't need the `ExternalEventContext` on your result object, unless you're using the information it contains when receiving the response?

Reply:
Possibly for the heartbeat logic, but that is coming as later feature. Will remove for now.